purchase_open_qty/views/purchase_view.xml

```diff
@@ -63,7 +63,7 @@
                 <filter
                     name="pending_qty_to_invoice"
                     string="Pending Qty to Bill"
-                    domain="[('qty_to_invoice','>',0.0)]"
+                    domain="[('qty_to_invoice','!=',0.0)]"
```

Comment:
Shouldn't you use a similar technique creating a computed field with `float_is_zero` (or `float_compare` comparison?

Reply:
In this case, the field is stored, so we can use the main field to check it, so I think that is not necesary to make another field

Reply:
The problem is not to be stored or not, but the rounding precision. A stored value of 0.0000001 will appear in this filter.

Reply:
I think, now this problem shouldn't appear